Assets/MixedRealityToolkit.Providers/XRSDK/Microsoft.MixedReality.Toolkit.Providers.XRSDK.asmdef

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "Microsoft.MixedReality.Toolkit.Providers.XRSDK",
+    "references": [
+        "GUID:a7a4c34601384f9389d5c4207f3c6331",
+        "GUID:92703082f92b41ba80f0d6912de66115",
+        "GUID:e40ba710768534012815d3193fa296cb"
+    ],
+    "includePlatforms": [],
+    "excludePlatforms": [],
+    "allowUnsafeCode": false,
+    "overrideReferences": false,
+    "precompiledReferences": [],
+    "autoReferenced": true,
+    "defineConstraints": [
```

Comment:
with this constraint, it will not be possible to add this provider to profiles on earlier versions of Unity and if present, the provider will appear to be "missing".

that could lead to customer confusion.

Reply:
I think that's okay. As a toolkit, we're in a bit of a unique situation where we have to straddle multiple Unity versions. I don't think that's a typical thing apps and other projects consuming MRTK do.

If they're building on Unity 2019.3, they get our (and Unity's) XR SDK types. If not, this assembly doesn't mean anything to them, and never will unless they update their project.